calcom/cal.com
# fix: profile banner delay for orgs + small DX chore

## What does this PR do?

When creating an org, the owner should see a profile banner to set up their profile once they land on the app. That was delayed since we didn't trigger rendering the corresponding component once things change.

<img width="1214" alt="image" src="https://github.com/calcom/cal.com/assets/467258/fd45483e-447d-478e-b0ad-0bd57741140f">

Also, when I was creating an org to replicate the issue and see the solution, noticed we sent the email with the code for lower envs either way and it was not needed since we introduced random totp codes for lower envs to stop relying on unstable dev mail services.

## Type of change


## How should this be tested?

Have orgs enabled locally, create one going to '/settings/organizations/new` and after org wizard, see the banner appear when you land in the app.

## Mandatory Tasks
